src/DynamoCoreWpf/UI/HandlingEventTrigger.cs

```diff
@@ -13,4 +19,113 @@ protected override void OnEvent(System.EventArgs eventArgs)
             base.OnEvent(eventArgs);
         }
     }
+
+    //forked from: http://www.mobilemotion.eu/?p=1822
+
+
+    public class EventToCommand
+    {
+        public static readonly DependencyProperty CommandProperty =
+            DependencyProperty.RegisterAttached("Commands", typeof(object), typeof(EventToCommand));
+
+        public static ICommand[] GetCommands(DependencyObject obj)
```

Comment:
these setters and accessors are not used when set from xaml - so unless someone else sets these dep properties from code behind it's unlikely to be used - but it's required for making an attached dep property.